test/e2e_node/util.go

```diff
@@ -82,7 +84,7 @@ func getNodeSummary() (*stats.Summary, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get current kubelet config")
 	}
-	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s:%d/stats/summary", kubeletConfig.Address, kubeletConfig.ReadOnlyPort), nil)
```

Comment:
i really wish we had a linter for this.

Reply:
;) https://github.com/kubernetes/kubernetes/pull/109344#issuecomment-1093403772